src/modeling/models/mmt_pretraining_model.py

```diff
@@ -17,20 +17,18 @@
 import tensorflow as tf
 from official.nlp.keras_nlp import layers
 
-import layers as mmt_layers
+from modeling import layers as mmt_layers
 
 
 class MmtPretrainingModel(tf.keras.Model):
   """Multimodal Transformer-based pretraining model.
 
-  Adds the masked language model (mlm), masked patch prediction (mpp) and
+  Adds the Masked Language Modeling (MLM), Masked Patch Prediction (MPP) and
   optional classification heads upon the transformer encoder.
 
   Args:
-    mmt_encoder: A transformer network. This network should output a
-      sequence output.
-    mpp_output_num_classes: The number of output classes for patch prediction.
-      It should be `(2 ** output_channel_bits ) ** 3`.
+    encoder: A transformer network. This network should output a
```

Comment:
Check grammar, "should output a sequence output" doesn't look grammatically correct. Also, missing period at the end.